modules/structs/org_type.go

```diff
@@ -27,7 +27,7 @@ var VisibilityModes = map[string]VisibleType{
 
 // IsPublic returns true if VisibleType is public
 func (vt VisibleType) IsPublic() bool {
-	return vt == VisibleTypePublic
+	return vt == VisibleTypePublic || vt == 0
```

Comment:
you dont have to since VisibleTypePublic is alias for int 0

Reply:
Removed